### ISSUE TITLE
RDF Schema Editting Issues

### ISSUE THREAD

**Issue by packet-rat** [OpenRefine/OpenRefine#1154]
Versions: Mac OSX 10.10.5, OpenRefine Version 2.6-rc.2 (1), Google Chrome Version 47.0.2526.80 (64-bit) [same behavior with any browser).

Many issues creating/editing RDF Schemas.  **Looking for general guidance** -- Linked Data schema editing behavior is inconsistent with any and all tutorials, books, etc.  For example resetting the RDF Schema for the Powerhouse Museum Data Set presents a window where the data overwrites the window and buttons cannot be easily reached (i.e., OK, Cancel)

![openrefine-schemaissue-1_1](https://cloud.githubusercontent.com/assets/3220193/18255243/afd0e65e-7375-11e6-9037-b49fb8422149.png)

**Comment by ettorerizza** [comment]
The Deri's RDF extension is outdated and doesn't work well with Open
Refine. Try with Google Refine.

Kind regards.

2016-09-05 20:40 GMT+02:00 Patrick Maroney <notifications@github.com>:

**Comment by akuckartz** [comment]
Is this the best source code to use as a foundation of a replacement of Deri's RDF extension: https://github.com/sparkica/rdf-extension ?

**Comment by packet-rat** [comment]
@ettorizza: Many thanks - seconding the question as to which rdf-extension to use with 2.6 (or is advice to revert to Google Refine for RDF related work?)

**Comment by ettorerizza** [comment]
Updating extensions depends on their developer. The version "Open Refine
2.6" of Deri's RDF extension  is in pre -release for two years ... (
https://github.com/fadmaa/grefine-rdf-extension/releases )
Its author was probably moved on. Personally, I use without problems three
versions of Open Refine ( 2.6 RC2, Google Refine 2.5 and LODRefine ). I
export my projects and i re-import them into one or another version based
on the specific work that I want to perform.

2016-09-06 0:35 GMT+02:00 Patrick Maroney <notifications@github.com>:

**Comment by packet-rat** [comment]
@ettorerizza :   Many thanks for the guidance.  I've had good success with OpenRefine and LODRefine by replacing the deprecated rdf-extension in the source distributions. 

I'm a "python guy" (primarily developing on Mac OSX) and looking for some general guidance on how you manage the multiple instantiations. For most scenarios I use Brew and Virtualenvwrapper to control what packages and external libraries are used for a given host environment executed instantiation.  Where required, I use Vagrant and Docker to further control environments (or suppprt things that need Linux or Windows variants). I also use pycharm vs. eclipse as my "goto" IDE.

I have run into issues with unintentional interactions between various OpenRefine and LODRefine instantiations on my base OSX system.  I've built the distributions from source (on Mac OSX).   I've edited the build.xml files to point (via relative references) to the new extensions.  I then clean and rebuild from source.  I then run them from command line in the given instantiations root directory.

 In the case of LODRefine, I eventually discovered it was actually using a deprecated rdf-extension from an OpenRefine instantiation.  Once I deleted the offending extension from OpenRefine, LODRefine used the "right" rdf-extension.

I could use Vagrant or Docker to isolate instantiations but prefer to run this from my base OSX system to leverage various development/debugging toola.
